### ISSUE TITLE
Custom Pick List / Lookup

### ISSUE THREAD

**Issue by stixoffire** [volkanceylan/Serenity#4270]
Implementation question..

Take for example I have a projects table with 2019-nnnn as their Name. I have some projects which are not yet real projects and I would like to be able to assign a pseudo project number such as PS19-nnnn.

I also have tasks that I must assign to either of these ..  (and this is where the custom lookup selection pick list would be - and I would need specifically the actual field and not the FK_Id) 

So basically a user can select an existing project or a pseudo project (or create new pseudo project - if it is not in the list) 

Should I just create a PseudoProject Table and do a join on the two tables and use that as a lookup ?
I would want the text value and not the Id in the task Column .

**Comment by Zahar661** [comment]
Difficult to write! There is 1 table projects, there a sign of pseudo or real project. and all the problems will be gone.

**Comment by JohnRanger** [comment]
Yeah - as @Zahar661 says: Make one project table and a flag field (e.g. a bit field / boolean) with which you flag a project to be a pseudo project or not.

Then you also easily can put a quick filter on it to only show pseudo or real projects - or both.

With kind regards,

John

**Comment by stixoffire** [comment]
@JohnRanger  Yes - problem is the real project table has special code to generate a Real Sequence number in the form IDYY-NNNN and I do not want to use those for any pseudo projects. 

Basically when we are evaluating projects for a customer it is not yet a real project - so this is why I had desired two tables - to keep them separate and track them separately - so I can know the time / resources (people)  that need to be applied to non-project projects and projects alike .  I guess I can hit the real table from another page and place the data in there as I like (perhaps using a different sequence number of say nnnnxxxx so I do not overwrite or duplicate (the field will allow it) ..  

Simply allow a pick list from the real projects table to add in  - like they do in the multi - fields - I think I will need to check into that code for the multi field select - I do not remember where that was to allow that -maybe I saw it in the Movie sample.
